### DISCUSSION TITLE
How can I know the resolution for my Ender3 V2?

### DISCUSSION THREAD

**Discussion by Gil80** [FormerLurker/ArcWelderPlugin#203] [Q&A]
These values:

Resolution:
0.05mm
Maximum Arc Radius:
1000000mm

Are set by default when I view it in Octoprint.
But how do I know if they are correct for my ender 3 v2?

Also, what would be a good benchmark print to test before enabling the plugin and after, to see if and how it affects quality?

**Comment by seantapscott** [comment]
Just use defaults. They're well researched and Marlin's firmware polygon
resolution defaults match up with arcwelder.

In terms of stuff to print, print stuff with curves.
